charts/yetibot/templates/secrets.yaml

```
@@ -1,3 +1,4 @@
+{{- if .Values.manage_secrets }}
```

[Helm recommends camel case](https://helm.sh/docs/chart_best_practices/values/#naming-conventions) for values.

```suggestion
{{- if .Values.manageSecrets }}
```